### ISSUE TITLE
Knacss repository is not the latest version and does not match the documentation page

### ISSUE THREAD

**Issue by BlueInt32** [raphaelgoetter/knacss-reborn#17]
Sorry about the rant, but even though I love knacss since I found out about it about a year ago, I think there is still a lack in documentation for typical front-end newcomers who want to use it. I mean many people will test knacss out with `npm i knacss --save` and trust that this will get the latest version, but it actually won't. In my opinion this single fact could lead to many people switching back to other css frameworks (or worse, Bootstrap !), thinking that knacss is simply poorly updated or documented. 

Indeed, there seems to be only [one npm package](https://www.npmjs.com/package/knacss) connected to [the "old" KNACSS github repository](https://github.com/alsacreations/KNACSS). But the current documentation code about autogrid simply does not work with the latest version of this repository.

Instead of `knacss` github repository being the old one and `knacss-reborn` being the new one, it should be `knacss-archive` being the old one and `knacss` being the latest one (or something along those lines). 

I spent about 1 hour to find out about this and I think I am a very patient guy. In fact this was because I relied solely on [this documentation page](https://www.knacss.com/doc.html#intro) which is not so clear about the mismatch.

I think that : 
- repositories should be renamed so that knacss is always the latest version
- there should be a single NPM package, pointing to the latest version of knacss
- some unmissable message like the one on [the home page of knacss](https://www.knacss.com/), also in [this documentation](https://www.knacss.com/doc.html#install) about the fact that there are 2 repositories
- for "previous knacss" users, some documentation on the version they are using

**Comment by raphaelgoetter** [comment]
You're absolutely right. I'll work on that ASAP.

> some unmissable message like the one on the home page of knacss, also in this documentation about the fact that there are 2 repositories

That should be done already :

<img width="776" alt="Capture d’écran 2021-01-25 à 10 01 13" src="https://user-images.githubusercontent.com/491029/105683759-49e97580-5ef4-11eb-90bf-6113a7afd1ec.png">

**Comment by simon-budin** [comment]
You are right, I finally saw this message on the very home page of knacss, but as I had a bookmark on [the documentation page](https://www.knacss.com/doc.html#install), this was not so clear. 
By the way I find it awesome that you respond so quickly ! 👍

**Comment by BlueInt32** [comment]
You are right, I finally saw this message on the very home page of knacss, but as I had a bookmark on the documentation page, this was not so clear.
By the way I find it awesome that you respond so quickly ! 👍

**Comment by raphaelgoetter** [comment]
KNACSS Reborn a finalement été ré-intégré à KNACSS sous forme de version majeure 8.0.x 

https://github.com/alsacreations/KNACSS

(Ce repo va donc être supprimé prochainement)
